postfix

```diff
@@ -58,4 +60,11 @@ fi
 ## run postfix
 /usr/sbin/postfix start-fg 2>&1 | rg --line-buffered -v "${LOG_FILTER_REGEX:-"^$"}" &
 
+# Reload config on USR1 signal
+function PostfixReload {
```

Comment:
https://google.github.io/styleguide/shellguide.html#s7.1-function-names 
- function names should be lowercase
- inconsistent usage of the `function` keyword compared to the other functions in this script (i.e (`shutdown()`)